compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1534,30 +1534,30 @@ void OMR::X86::TreeEvaluator::genArithmeticInstructionsForOverflowCHK(TR::Node *
          case TR::badd:
          case TR::sadd:
          case TR::iadd:
-            addMulAnalyser.integerAddAnalyserWithExplicitOperands(node, operand1, operand2, op, TR::InstOpCode::bad, needsEflags);
+            addMulAnalyser.integerAddAnalyserWithExplicitOperands(node, operand1, operand2, op, TR::InstOpCode::int3, needsEflags);
```

Comment:
For the opcodes passed into the "analyser" functions (not just here, but for all of them), why did you pick `int3` vs. `ud2`?

Reply:
Not sure why I did that, but it should be ud2